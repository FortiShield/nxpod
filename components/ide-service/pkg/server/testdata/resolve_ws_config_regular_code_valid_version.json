{
  "context": "{\"isFile\":false,\"path\":\"\",\"title\":\"nxpkg/empty \",\"revision\":\"\",\"repository\":{\"cloneUrl\":\"https://github.com/nxpkg/empty.git\",\"host\":\"github.com\",\"name\":\"empty\",\"owner\":\"nxpkg\",\"private\":false},\"normalizedContextURL\":\"https://github.com/nxpkg/empty\",\"checkoutLocation\":\"empty\"}",
  "ide_settings": "{\"settingVersion\":\"2.0\",\"defaultIde\":\"code\",\"useLatestVersion\":false, \"pinnedIDEversions\":{\"code\":\"1.86.2\"}}",
  "workspace_config": "{\"ports\":[],\"tasks\":[],\"image\":\"docker.io/gitpod/workspace-full:latest\",\"_origin\":\"default\"}"
}
