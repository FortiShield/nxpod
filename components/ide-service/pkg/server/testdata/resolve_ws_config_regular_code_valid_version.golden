{
    "Resp": {
        "envvars": [
            {
                "name": "NXPOD_IDE_ALIAS",
                "value": "code"
            }
        ],
        "supervisor_image": "ghcr.io/nxpkg/build/supervisor:commit-ff38b98b7dde4929159bcaeec68d178898dc2139",
        "web_image": "ghcr.io/nxpkg/build/ide/code:commit-d86f6aa033943c9650d06339915e68063b0cf142",
        "ide_settings": "{\"defaultIde\":\"code\"}\n"
    },
    "Err": ""
}
